src/controllers/list.html

```diff
@@ -48,7 +48,7 @@
                 <span class="material-icons more_vert"></span>
             </button>
         </div>
-        <div is="emby-itemscontainer" class="vertical-wrap itemsContainer centered" style="padding: 0 0 10vh 0;">
+        <div is="emby-itemscontainer" class="vertical-wrap itemsContainer centered">
```

Comment:
Agree with this. Also I would add mentioned `padded-bottom-page` to `pageContainerTopPadding` element as above for the same purpose.